packages/xstate-react/test/useMachine.test.tsx

```diff
@@ -160,33 +162,34 @@ describeEachReactMode('useMachine (%s)', ({ suiteKey, render }) => {
     render(<Test />);
   });
 
-  it('should merge machine context with options.context', () => {
-    const testMachine = createMachine<{ foo: string; test: boolean }>({
-      context: {
-        foo: 'bar',
-        test: false
-      },
-      initial: 'idle',
-      states: {
-        idle: {}
-      }
-    });
-
-    const Test = () => {
-      const [state] = useMachine(testMachine, {
-        context: { test: true }
-      });
-
-      expect(state.context).toEqual({
-        foo: 'bar',
-        test: true
-      });
-
-      return null;
-    };
-
-    render(<Test />);
-  });
+  // TODO: make new test with input (not related to options.context)
+  // it('should merge machine context with options.context', () => {
+  //   const testMachine = createMachine<{ foo: string; test: boolean }>({
+  //     context: ({ input }) => ({
+  //       foo: 'bar',
+  //       test: input.test ?? false
+  //     }),
+  //     initial: 'idle',
+  //     states: {
+  //       idle: {}
+  //     }
+  //   });
+
+  //   const Test = () => {
+  //     const [state] = useMachine(testMachine, {
+  //       input: { test: true }
+  //     });
+
+  //     expect(state.context).toEqual({
+  //       foo: 'bar',
+  //       test: true
+  //     });
+
+  //     return null;
+  //   };
+
+  //   render(<Test />);
+  // });
```

Comment:
That feature will just have a completely new shape - so this particular test isn't that applicable to the post-input world, right? Perhaps this can just be removed from here already in this PR.